handlers/handlers.go

```diff
@@ -246,8 +249,51 @@ func HandleNoteApiRequest(
 
 		fmt.Fprint(responseWriter, string(notesInJson))
 
+	case http.MethodPost:
+		type NoteForm struct {
+			Content  string `json:"content"`
+			Category string `json:"category"`
+		}
```

Comment:
I think these fields need to go in two separate api calls. What is category when none is selected?

Reply:
Its empty string as you can see in line 277. To continue down your idea, saving a new note will potentially be two api calls? insert note and insert category? That's do-able.

Reply:
Yeah I was imagining two distinct api calls. It's the type of thing you squish together when you're looking to improve performance, but i see no reason to be optimizing for performance before there's an issue